ls1intum/Artemis
# `General`: Enhance exercise buttons with exercise type icons

### Checklist
#### General



#### Client


### Motivation and Context
Stephan suggested adding exercise-type icons for the Create/Import Exercise buttons in the exercise group management view to improve UX. 

### Description
These icons are already displayed on mobile, when the button label is not displayed anymore. This PR displays them also on desktop after the label.

Edit: Same goes for the course exercise create/import buttons and the order of create and import buttons is unified to first the "Create" Button and then the "Import" Button.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Exam with 1 Exercise Group
- View the control section there on mobile and desktop
- Verify that the exercise-type icons are displayed on both

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots



Desktop:
![image](https://github.com/ls1intum/Artemis/assets/78978542/1bdd74de-94b7-4805-8bea-6678781a19aa)

Mobile:
![image](https://github.com/ls1intum/Artemis/assets/78978542/c1bca1af-c444-4db6-9f5e-9404259ad649)

course exercises:
![image](https://github.com/ls1intum/Artemis/assets/78978542/c4c3c2fa-d7cb-46e1-861c-d4fe82f3e70c)
